docs/contributing/03-stories/story-32.md

```diff
@@ -0,0 +1,107 @@
+# User Story 31 - Wing Platforms
+
+Marcus works on a Platform Engineering for a large company that has multiple internal teams all building applications that get deployed into the cloud. 
+
+Recently, his company decided to start investigating adopting Winglang as the main language for their internal teams. However, his company's Cloud Center of Excellence (CCoE) 
+has a few concerns about adopting Winglang. Primarily, they are concerned about how they will still be able to enforce company best practices and standards across all the teams,
+if Wing infers the infrastructure from the code.
+
+Marcus, has taken up the torch to investigate how they can conform the Winglang code written by the internal teams to the company standards. He has stumbled upon the concept of
+Wing Platforms and is now a few energy drinks deep into a late night investigation.
+
+The Wing documentation on Custom Platforms depicts an exposed `interface` called `IPlatform` which defines the methods that a Platform may implement. 
+
+```typescript
+export interface IPlatform {
+  newInstance?(type: string, scope: Construct, id: string, props: any): any;
```

Comment:
missing `model` here, no?

Reply:
Yes you are right, I wrote most of the draft before our chat 😅